### PR TITLE
added Samurai theme

### PR DESCRIPTION
**Pull request by no3l0815**
added a new theme

### CODE REVIEW

**Comment by no3l0815** [comment]
It's my first pull request so please text me if i did something wrong.

**Comment by morpheusthewhite** [comment]
There is no README, please add it to the folder as explained here https://github.com/morpheusthewhite/spicetify-themes#contributions

**Comment by no3l0815** [comment]
I added now the README

File: Samurai/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+#Samurai
+
+##Screenshots 
+
+![Samurai](./samuraitheme.png)
+
+#more
```
Comment (morpheusthewhite) [inline Samurai/README.md:8]: Correct the headers and check the preview https://github.com/no3l0815/spicetify-themes/tree/master/Samurai

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]

**Review by no3l0815 — COMMENTED** [review]
everythings ok

**Comment by morpheusthewhite** [comment]
Nope, spaces are needed between the #s and text or they will not be shown correctly (same for ##)

**Review by morpheusthewhite — CHANGES_REQUESTED** [review]
Please correct the README

**Comment by no3l0815** [comment]
i changed it now

**Pull request merged**